src/structlog/dev.py

```diff
@@ -538,6 +546,7 @@ def __init__(  # noqa: PLR0912
         event_key: str = "event",
         timestamp_key: str = "timestamp",
         columns: list[Column] | None = None,
+        skip_level_padding: bool = False,
```

Comment:
ugh `pad_event` is a terrible name in hindsight since it takes an int, but let's make this one `pad_level: bool = True` anyway

Reply:
Fixed